.tav.yml

```diff
@@ -94,18 +94,27 @@ redis:
 # subset (the first, plus then the latest in each major.minor).
 ioredis-v2-v4:
   name: ioredis
-  versions: '2.0.0 || 2.0.1 || 2.1.0 || 2.2.0 || 2.3.1 || 2.4.3 || 2.5.0 || 3.0.0 || 3.1.4 || 3.2.2 || >3.2.2 <4'
+  versions: '2.0.0 || 2.0.1 || 2.1.0 || 2.2.0 || 2.3.1 || 2.4.3 || 2.5.0 || 3.0.0 || 3.1.4 || ^3.2.2'
```

Comment:
Note for reviewer: this caret version is added by the script for all modes. We should decide if
- we use caret or range for all modes
- caret/range depends on the selected mode

Reply:
Perhaps prefer the `>3.2.2 <4` style? At least in the case where you want to catch a new major release, it can handle it. For example:   `^1.2.3` will not catch a possible new `2.x` release. However only can write `>=1.2.3` or `>=1.2.3 < 2` or `>=1.2.3 <3` to specify a larger range as needed.

Reply:
Caret if fine though unless we run into a case where we *need* the "next major" range.